### ISSUE TITLE
Document how to add a new grader

### ISSUE THREAD

**Issue by tansaku** [saasbook/rag#63]
I was trying to add a new grader: https://github.com/saasbook/rag/tree/62_support_legacy_assignment but it wasn't automatically included.  The grader choked saying that HerokuWithLoginRspecGrader was unknown. 

I guess that the new grader just needs to be required somewhere?  Perhaps we should set it so that all rb files in the grader directory are automatically pulled in?

**Comment by zhangaaron** [comment]
Yes will work on that. By the way are we still using the AV PivotalTracker for tracking issues because it doesn't appear maintained, or GH Issues? I have my own PivotalTracker for issues but I think it would be nice if we could merge.

**Comment by tansaku** [comment]
hi @zhangaaron sorry - didn't see your post here - my bad - see my email about using github issues (and possibly waffle) for managing issues.  I haven't touched the AV Pivotal Tracker for autograders in a while, and if you haven't been using it it's probably pretty out of date ...
